ecs.php

```diff
@@ -13,7 +15,11 @@
 
     $parameters->set(
         Option::SKIP,
-        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]
+        [
+            'SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php'],
+            OperatorSpacingSniff::class => null,
+            BinaryOperatorSpacesFixer::class => null,
```

Comment:
What about adding [this](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5495#issuecomment-777187487) directly to lmc/coding-standard?

Reply:
AFAIK it should be handled properly in csfixer 3.6.0